.github/workflows/bump_submodule.yml

```diff
@@ -0,0 +1,58 @@
+name: Bump rocm-libraries submodule
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 11 * * *"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  bump-submodules:
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c # v6.0.0
+        with:
+          python-version: '3.12'
+
+      - name: Run bump submodules script
+        run: |
+          git config --global user.name "therockbot"
+          git config --global user.email "therockbot@amd.com"
+
+          DATE=$(LC_ALL=C date +%Y%m%d)
+          python3 ./build_tools/bump_submodules.py \
+            --components rocm-libraries \
+            --push-branch \
+            --branch-name github-actions/bump-rocm-libs-$DATE
+
+      - name: Generate GitHub App token
+        id: generate-token
+        uses: actions/create-github-app-token@b36a0b4db3c4ce0a0d11a8d6c20d7b19d4c89d2f # v2.1.0
+        with:
+          app-id: ${{ secrets.PULL_REQUEST_APP_ID }}
+          private-key: ${{ secrets.PULL_REQUEST_APP_KEY }}
+
+      - name: Create pull request
```

Comment:
Can you share a PR this created so we can inspect it? Did test here or on your fork somewhere? I don't see any notes in the PR description to that effect.